draft-irtf-cfrg-bls-signature.md

```diff
@@ -771,17 +771,24 @@ Outputs:
 Precondition: n >= 1, otherwise return INVALID.
 
 Procedure:
-1.  R = signature_to_point(signature)
-2.  If R is INVALID, return INVALID
-3.  If signature_subgroup_check(R) is INVALID, return INVALID
-4.  C1 = 1 (the identity element in GT)
-5.  for i in 1, ..., n:
-6.      If KeyValidate(PK_i) is INVALID, return INVALID
-7.      xP = pubkey_to_point(PK_i)
-8.      Q = hash_to_point(message_i)
-9.      C1 = C1 * pairing(Q, xP)
-10. C2 = pairing(R, P)
-11. If C1 == C2, return VALID, else return INVALID
+1  compute the l distinct messages m_1, m_l
+2. Aggregate the public keys of the same message to l sets of public keys QK_1_1, ...,QK_1_m, QK_2_1,..., QK_2_p, ..., QK_l_1,...,QK_l_q   
+3.  R = signature_to_point(signature)
+4.  If R is INVALID, return INVALID
+5.  If signature_subgroup_check(R) is INVALID, return INVALID
+6.  C1 = 1 (the identity element in GT)
```

Comment:
not sure identity for GT is defined. may be we need to add an entry [here](https://github.com/cfrg/draft-irtf-cfrg-bls-signature/blob/98bc7b61e4d5bb194c6f3df042747fe0ca6c1016/draft-irtf-cfrg-bls-signature-04.txt#L416)

Reply:
@zhenfeizhang this might be a valid comment but it was already part of the current version of the draft